server/src/main/java/org/spine3/server/stand/StandStorage.java

```diff
@@ -69,4 +72,24 @@ protected StandStorage(boolean multitenant) {
      */
     public abstract ImmutableCollection<EntityRecord> readAllByType(TypeUrl type,
                                                                     FieldMask fieldMask);
+
+    /**
+     * The {@code StandStorage} is specified not to satisfy the {@link EntityQuery}.
```

Comment:
As stated, the `readAll(EntityQuery, FieldMask)` operation is not supported in the `StandStorage` since there are no Entity Columns in `StandStorage`(#400). Instead, this method calls `readAll(FieldMask)`. 

That's why the `{@inheritDoc}` is omitted in this case (since we totally override the behavior).

As soon as `StandStorage` supports Entity Columns, this particular issue will also be resolved.